src/app/pages/article/article.js

```diff
@@ -0,0 +1,91 @@
+import LoadingView from '~/helpers/backbone/loading-view';
+import {template as strips} from '~/components/strips/strips.hbs';
+import {props} from '~/helpers/backbone/decorators';
+import {template} from './article.hbs';
+import Articles from '../blog/articles/articles';
+import PageModel from '~/models/pagemodel';
+
+@props({
+    template: template,
+    css: '/app/pages/article/article.css',
+    templateHelpers: {strips},
+    regions: {
+        articles: '.articles'
+    }
+})
+export default class Article extends LoadingView {
+
+    onLoaded() {
+        super.onLoaded();
+        this.el.querySelector('.article.page').classList.remove('hidden');
+        document.body.classList.remove('no-scroll');
+    }
+
+    onRender() {
+        document.body.classList.add('no-scroll');
+        this.el.querySelector('.article.page').classList.add('hidden');
+
+        let slug = window.location.pathname.replace(/.*\//, '');
+
+        new PageModel().fetch({
+            data: {
+                type: 'news.NewsArticle',
+                slug: slug
+            }
+        }).then((data) => {
+            new PageModel().fetch({url: data.pages[0].meta.detail_url}).then((articleData) => {
+                document.title = `${articleData.title} - Openstax`;
+                for (let el of this.el.querySelectorAll('[data-id]')) {
+                    let key = el.dataset.id;
+
+                    if (key in articleData) {
+                        el.innerHTML = articleData[key];
+                    }
+                }
+
+                let articleImage = articleData.article_image || 'http://placehold.it/1400x600';
+
+                this.el.querySelector('.hero').setAttribute('style', `background-image:url(${articleImage})`);
+
+                let d = new Date(articleData.date).toDateString().split(' ');
+                let formatDate = `${d[1]} ${d[2]}, ${d[3]}`;
+
+                this.el.querySelector('[data-id="date"]').innerHTML = formatDate;
+                let newTags = [];
+
+                for (let tag of articleData.tags) {
+                    let spanObj = `<span>${tag}</span>`;
+
+                    newTags += spanObj;
+                }
+
+                this.el.querySelector('.tags').innerHTML = newTags;
+
+                /* disabling the following eslint requirements due to disqus_config requirements */
+                /* eslint no-unused-vars: 0 */
+                /* eslint camelcase: 0 */
+                let disqus_config = () => {
```

Comment:
Is this something that will be used in the future when we hook up Disqus for comments?